### PR TITLE
feat(game-refs/veh-refs): add vehicle-references and vehicle flags

### PR DESCRIPTION
**Pull request by spacevx**
* Moved vehicles data into the vehicle-references section
* Added a vehicle flags page

Related to https://github.com/citizenfx/fivem/pull/2780
Was tested by building the docs

### CODE REVIEW

File: content/docs/game-references/vehicle-references/_index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Vehicle references
+weight: 700
```
Comment (AvarianKnight) [inline content/docs/game-references/vehicle-references/_index.md:3]:
```suggestion
weight: 740
```
This shouldn't be before controls.

File: content/docs/game-references/vehicle-references/vehicle-flags.md [hunk 2]
```diff
@@ -0,0 +1,931 @@
...
+        color: #555; /* Medium gray for paragraphs to soften the contrast */
+    }
+
+    .flag .build {
+        font-style: italic;
+        color: #888; /* Softer gray for build info to distinguish it subtly */
+    }
+</style>
+
+<div class="flags">
```
Comment (AvarianKnight) [inline content/docs/game-references/vehicle-references/vehicle-flags.md:43]: Why not use a markdown table here?

**Review by AvarianKnight — CHANGES_REQUESTED** [review]

File: content/docs/game-references/vehicle-references/_index.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: Vehicle references
+weight: 700
+layout: single
+---
+
+Reference lists of vehicle data are in this category
+
+- [Vehicle colours](/docs/game-references/vehicle-references/vehicle-colours)
```
Comment (PsychoShock) [inline content/docs/game-references/vehicle-references/_index.md:9]:
```suggestion
- [Vehicle colors](/docs/game-references/vehicle-references/vehicle-colors)
```

Docs use AmE so you need to remove `u` from `colours`. Don't forget to change the page file too.

**Review by PsychoShock — CHANGES_REQUESTED** [review]

**Comment by ahcenezdh** [comment]
@AvarianKnight should be good

File: content/docs/game-references/input-mapper-parameter-ids/_index.md [hunk 4]
```diff
@@ -1,64 +1,64 @@
...
-[mouse_normalized]: /docs/game-references/input-mapper-parameter-ids/mouse_normalized
-[mouse_wheel]: /docs/game-references/input-mapper-parameter-ids/mouse_wheel
-[pad_analogbutton]: /docs/game-references/input-mapper-parameter-ids/pad_analogbutton
-[pad_axis]: /docs/game-references/input-mapper-parameter-ids/pad_axis
-[pad_debugbutton]: /docs/game-references/input-mapper-parameter-ids/pad_debugbutton
-[pad_digitalbutton]: /docs/game-references/input-mapper-parameter-ids/pad_digitalbutton
-[pad_digitalbuttonany]: /docs/game-references/input-mapper-parameter-ids/pad_digitalbuttonany
-[touchpad_absolute_axis]: /docs/game-references/input-mapper-parameter-ids/touchpad_absolute_axis
-[touchpad_centered_axis]: /docs/game-references/input-mapper-parameter-ids/touchpad_centered_axis
+---
```
Comment (AvarianKnight) [inline content/docs/game-references/input-mapper-parameter-ids/_index.md:1]: Why does this file have any changes?

Comment (ahcenezdh) [reply content/docs/game-references/input-mapper-parameter-ids/_index.md:1]: Probably my vscode

Comment (ahcenezdh) [reply content/docs/game-references/input-mapper-parameter-ids/_index.md:1]: (Yes those are whitespaces it's my vscode who's messing with whitespaces)

Comment (AvarianKnight) [reply content/docs/game-references/input-mapper-parameter-ids/_index.md:1]: Can you remove the changes and then force push?

Comment (ahcenezdh) [reply content/docs/game-references/input-mapper-parameter-ids/_index.md]: Should be good

**Comment by AvarianKnight** [comment]
Looks good to me, thanks!

**Review by AvarianKnight — APPROVED** [review]

**Pull request merged**